lib/process-services-cloud/src/lib/services/web-socket.service.ts

```diff
@@ -135,9 +141,25 @@ export class WebSocketService {
                 max: Number.POSITIVE_INFINITY,
                 jitter: true
             },
-            attempts: {
-                max: 5,
-                retryIf: (error) => !!error
+            attempts: (count: number, _operation: Operation, error: any) => {
+                if (!error) {
+                    return false;
+                }
+
+                const isUnauthorizedError =
```

Comment:
I think we can reuse this part to actually catch the error from network when it happens and then try to refresh the token. Same with the retry mechanism instead of having a plain number we can use the retryIf